enterprise/backend/test/metabase_enterprise/serialization/api_test.clj

```diff
@@ -205,3 +207,11 @@
         ;; in `api/on-response!`, so maybe add some Thread/sleep here.
         (is (= known-files
                (set (.list (io/file api.serialization/parent-dir)))))))))
+
+(deftest find-serialization-dir-test
+  (testing "We are able to find serialization dir even in presence of various hidden dirs"
+    (let [dst (io/file api.serialization/parent-dir (u.random/random-name))]
+      (.mkdirs (io/file dst "._hidden_dir"))
```

Comment:
I think we're missing a test case where it's not a "dotdir", but still lacks any of the expected sub-directories.